.github/workflows/build_docker_images_for_ci.yml

```diff
@@ -11,7 +11,7 @@ on:
       - '.github/workflows/build_docker_images_for_ci.yml'
```

Comment:
maybe here it should be added the new script? @philbucher

Reply:
ok, you just did the change :)

Reply:
we had the same thought, I already did it :)